x-pack/test/functional/apps/security/management.js

```diff
@@ -146,10 +146,12 @@ export default function ({ getService, getPageObjects }) {
           await PageObjects.security.assignRoleToUser('kibana_dashboard_only_user');
           await PageObjects.security.assignRoleToUser('logstash-readonly');
 
-          await PageObjects.security.clickSaveEditUser();
+          await PageObjects.security.clickSaveCreateUser();
 
           await PageObjects.settings.navigateTo();
-          await testSubjects.click('users');
+          await testSubjects.click('roles');
+          await testSubjects.click('tablePaginationPopoverButton');
+          await testSubjects.click('tablePagination-100-rows');
           await PageObjects.settings.clickLinkText('kibana_dashboard_only_user');
           const currentUrl = await browser.getCurrentUrl();
           expect(currentUrl).to.contain(EDIT_ROLES_PATH);
```

Comment:
This test is 'Can navigate to edit role section from users page'.

It looks like it's trying to exercise the following:
1. Create a user with the 'kibana_dashboard_only_user' role
2. From the Users page, click the 'kibana_dashboard_only_user' role
3. Assert that our URL now contains the Edit Roles path (`security/roles/edit`)

You've changed this test to:

1. Create a user with the 'kibana_dashboard_only_user' role
2. Navigate to the Roles page, and expand the per-page view to 100 rows
3. From the Roles page, click the 'kibana_dashboard_only_user' role
3. Assert that our URL now contains the Edit Roles path (`security/roles/edit`)

Is there any reason this didn't work before changing lines 152-154? It seems like the only change needed in this section is line 149.

Reply:
reverted by clicking on `logstash-readonly` role instead of 'kibana_dashboard_only_user'  which is a deprecated role and will be removed in 8.0